app/views/layouts/application.html.erb

```diff
@@ -42,16 +42,14 @@
         <%= content_for(:before_content) %>
 
         <% if content_for(:content).present? || content_for(:sidebar).present? %>
-          <div class="row">
-            <% if content_for(:sidebar).present? %>
-              <div class="col-md-2 col-sm-3 sidebar sidebar-left sidebar-animate sidebar-md-show">
-                <%= content_for(:sidebar) %>
-              </div>
-            <% end %>
-
-            <div class="<%= content_for(:content_size).present? ? content_for(:content_size) : 'col-xs-12' %>">
-              <%= render partial: "common/contents" %>
+          <% if content_for(:sidebar).present? %>
+            <div class="col-md-2 col-sm-3 sidebar sidebar-left sidebar-animate sidebar-md-show">
```

Comment:
This looks like it is now a `col` within a `col` instead of `row` `col` `row` `col`... I fell like this is not correct bootstrap code.

Reply:
It might have some off rendering where it won't align properly because of the nested `col`.